File: switch/app/src/main/java/algonquin/cst0459/aswitch/MainActivity.java
package algonquin.cst0459.aswitch;



import android.os.Bundle;

import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.Switch;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private Switch sw1, sw2;
    private Button btnGet;
    private ImageButton addBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Initialize switches and button
        sw1 = findViewById(R.id.switch1);
        sw2 = findViewById(R.id.switch2);
        btnGet = findViewById(R.id.getBtn);
        addBtn = findViewById(R.id.addBtn);

        btnGet.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String str1, str2;
                if (sw1.isChecked()) {
                    str1 = sw1.getTextOn().toString();
                } else {
                    str1 = sw1.getTextOff().toString();
                }

                if (sw2.isChecked()) {
                    str2 = sw2.getTextOn().toString();
                } else {
                    str2 = sw2.getTextOff().toString();
                }

                Toast.makeText(getApplicationContext(), "Switch1 - " + str1 + "\nSwitch2 - " + str2, Toast.LENGTH_SHORT).show();
            }
        });

        addBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(), "ImageButton Clicked", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: pratik/app/src/main/java/algonquin/cst0459/pratik/MainActivity.kt
import android.os.Bundle
import android.widget.Button
import android.widget.Switch
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var sw1: Switch
    private lateinit var sw2: Switch
    private lateinit var btnGet: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Initialize switches and button
        sw1 = findViewById(R.id.switch1)
        sw2 = findViewById(R.id.switch2)
        btnGet = findViewById(R.id.getBtn)

        btnGet.setOnClickListener {
            val str1 = if (sw1.isChecked) sw1.textOn.toString() else sw1.textOff.toString()
            val str2 = if (sw2.isChecked) sw2.textOn.toString() else sw2.textOff.toString()

            Toast.makeText(
                applicationContext,
                "Switch1 -  $str1 \nSwitch2 - $str2",
                Toast.LENGTH_SHORT
            ).show()
        }
    }
}

}
}